control-plane-operator/hostedclusterconfigoperator/controllers/inplaceupgrader/inplaceupgrader.go

```diff
@@ -224,9 +226,14 @@ func (r *Reconciler) reconcileInPlaceUpgrade(ctx context.Context, nodePoolUpgrad
 	}
 
 	// Find nodes that can be upgraded
-	// The drain is handled separately, such that if the node is awaiting drain, nothing will happen here.
-	// TODO (jerzhang): add logic to honor maxUnavailable/maxSurge
-	nodesToUpgrade := getNodesToUpgrade(nodes, targetConfigVersionHash, 1)
+	maxunavail := 1
```

Comment:
nit: can we be consistent with [mixed-caps](https://go.dev/doc/effective_go#mixed-caps) naming here as in `nodePoolAnnotationMaxUnavailable` i.e `maxUnavailable`, `maxUnavailableAnnotation`...